test/quantization/core/test_quantized_op.py

```diff
@@ -3740,6 +3740,117 @@ def test_qlinear_unpack(self, W, use_channelwise):
             np.testing.assert_equal(
                 W_q.q_zero_point(), W_q_origin.q_zero_point())
 
+    """Tests the correctness of the quantized linear_leaky_relu op."""
+    @given(batch_size=st.integers(1, 4),
+           input_channels=st.integers(16, 32),
+           output_channels=st.integers(4, 8),
+           use_bias=st.booleans(),
+           use_multi_dim_input=st.booleans(),
+           use_channelwise=st.booleans())
+    @override_qengines
+    def test_qlinear_leaky_relu(self, batch_size, input_channels, output_channels, use_bias,
```

Comment:
this is really long, can you add some comments to describe overall what we are testing here?

Reply:
This test is based on `test_qlinear` just above: https://github.com/pytorch/pytorch/blob/299fbbccec68f9e8e0fa19a1066a8e179e9ab633/test/quantization/core/test_quantized_op.py#L3467
So, the structure and logic of the 2 UTs are the same, as well as usages of utils functions. This may also answer your questions below. Maybe we can extract some common parts of the two tests in the same test case `TestQuantizedLinear` https://github.com/pytorch/pytorch/blob/299fbbccec68f9e8e0fa19a1066a8e179e9ab633/test/quantization/core/test_quantized_op.py#L3457